heron/tracker/src/python/main.py

```diff
@@ -205,13 +208,30 @@ def main():
   else:
     configure_logging(logging.INFO)
 
+  # set Tornado global option
+  define_options(namespace['port'], namespace['config_file'])
+
+  # create Tornado application
+  application = Application()
+
+  # pylint: disable=unused-argument
+  # SIGINT handler:
+  # 1. stop all the running zkstatemanager and filestatemanagers
+  # 2. stop the Tornado IO loop
+  def signal_handler(signum, frame):
```

Comment:
`signal_handler` in `heron-tracker` has to perform `application.stop()`, but `heron-ui` does not have to.